bootcd/genesis.ks

```diff
@@ -54,6 +54,7 @@ tmux
 traceroute
 util-linux-ng
 vim-minimal
+bind-utils
```

Comment:
You don't like alphabetical ordering?

BTW, I plan to add ntpdate and run it during genesis bootstrapping since verifying SSL certs when the date is way off will fail if the cert hasn't "always" been valid.